### ISSUE TITLE
Spoke Core Contributors meta issue

### ISSUE THREAD

**Issue by ibrand** [ProgressiveCoders/Spoke#1205]
This meta issue forms the agreement for folks who meet the criteria to become a member of the [Spoke Core Contributors Group](https://github.com/MoveOnOrg/Spoke/wiki/Spoke-Access-Groups) to affirm before getting access to the group.

**As a Spoke core contributor, I agree to the following:**
- I will not merge to main
- I will not merge to stage-main
- I will not use labels: “testing on stage”, “testing in production” related to mergability
- I have read and affirm the [code of conduct](https://github.com/MoveOnOrg/Spoke/blob/main/CODE_OF_CONDUCT.md)
- If I break code of conduct, or break any of the other agreements, I understand that I might lose Core Contributor status

**Comment by azuzunaga** [comment]
I agree!
🎉

**Comment by filafb** [comment]
I agree!!

**Comment by joemcl** [comment]
I agree. Also would be good to clarify what contributors *can* do, not just what we can't :) . cc: @deasterdaywfp @lperson @bdatkins

**Comment by deasterdaywfp** [comment]
Couldn't agree more!

**Comment by joemcl** [comment]
@lperson pls take a look at ^^^ and if you're good, would love to get you in as a Contributor here, "officially" :) .

**Comment by ibrand** [comment]
Hey everyone! I'm interested in starting an auto-assign rotation for pull requests to make reviewing PRs move a bit faster and also create a real and sustainable review culture that helps keep code quality high. This would also help a lot with our ability to bring in new folks because fast review turnaround dramatically increases contributor rate of return.

The way this would work, is that we'd implement [this bot that auto-assigns a reviewer to a PR](https://probot.github.io/apps/auto-assign/) and then some (or all) of us would be in the rotation of reviewers with the ability to opt out. If we were on vacation or away from the project for any reason, [I'd build a feature for this bot to allow us to not be included in the rotation for that period of time](https://github.com/kentaro-m/auto-assign/issues/108)

In preparation, I've written up a doc that is included in the next merge round on how to code review.

I could see 3 options here:
1. Part of being a core-contributor is that you’re automatically added to the list of reviewers (which you can then opt out of)
2. If you’d like, you can make a PR to the config file and add yourself to the list of contributors
3. No auto-assigning at all.

For option 1, react to this with 🎉 
For option 2, react with 👍 
For option 3, react with 👎 

And if you have any thoughts or questions, share them here!

**Comment by ibrand** [comment]
@joemcl @azuzunaga @filafb

**Comment by filafb** [comment]
Option 1 is the way to go, but not sure how much work it means. We can definitely start there, but review it after a few months or so.

**Comment by joemcl** [comment]
@ibrand I voted for 1, but need a way for the randomly-assigned person to add in/recruit other reviewers if the randomly assigned person (say, me) does not feel competent to review the code in the PR but thinks another person is. Also I'd like a way to be able to ask folks like @bchrobot @harpojaeger etc to be able to review PRs here even if they haven't yet signed up per above.

**Comment by ibrand** [comment]
@joemcl I did some reading about this and it looks like only folks explicitly added with read access can be assigned PRs. I'm going to add a handful of folks right now with read access so that you can re-assign PRs that get assigned to you to someone appropriate. That said, if you don't feel qualified to review PRs more generally, option 1 does allow you to opt out of being in the review cycle!

**Comment by lperson** [comment]
I agree to the terms and I opt in.

**Comment by bdatkins** [comment]
I agree and I'm in!

**Comment by JeremyParker** [comment]
I agree to these terms and opt in.

**Comment by lperson** [comment]
I agree to these terms and opt in.

**Comment by jeffm2001** [comment]
I agree!

**Comment by hiemanshu** [comment]
I agree! And I'm in!
